src/components/molecules/MyContractsTable/index.tsx

```diff
@@ -0,0 +1,160 @@
+import { Grid, Table, TableBody, TableCell, TableHead, TableRow } from "@mui/material";
+import theme from "../../../utils/theme";
+import Typography from "../../atoms/Typography";
+import { IMyContractsTableColumnLabels, IMyContractsTableDataDetail } from "../../../utils/myContracts";
+
+export interface IMyContractsTableProps {
+    tableColumns: IMyContractsTableColumnLabels[];
+    tableData: IMyContractsTableDataDetail[];
+};
+
+const MyContractsTable = ({ tableColumns, tableData }: IMyContractsTableProps) => {
```

Comment:
you have have the molecule name as Table instead of MyContractsTable as molecules are reusable in other parts of code. So name must be generic.